src/bci_build/package/__init__.py

```diff
@@ -716,13 +607,35 @@ def build_release(self) -> str | None:
             return "35"
         return None
 
+    @property
+    def distribution_base_name(self) -> str:
+        if self.os_version.is_tumbleweed:
+            return "openSUSE Tumbleweed"
+        elif self.os_version.is_ltss:
+            return "SLE LTSS"
+        elif self.os_version.is_sle15:
+            return "SLE"
+        elif self.os_version.value == OsVersion.BASALT.value:
+            return "Basalt Project"
+
+        raise NotImplementedError(f"Unknown os_version: {self.os_version}")
```

Comment:
it can not because the implementation is overriden in `DevelopmentContainer` and `ApplicationStackContainer`. I would have to pass "self" then to os_version which I consider super weird.